app/protocol/flows/v4/blockrelay/handle_pruning_point_and_its_anticone_requests.go

```diff
@@ -57,8 +58,73 @@ func HandlePruningPointAndItsAnticoneRequests(context PruningPointAndItsAnticone
 				return err
 			}
 
+			windowSize := context.Config().NetParams().DifficultyAdjustmentWindowSize
+			daaWindowBlocks := make([]*externalapi.TrustedDataDataDAAHeader, 0, windowSize)
+			daaWindowHashesToIndex := make(map[externalapi.DomainHash]int, windowSize)
+			trustedDataDAABlockIndexes := make(map[externalapi.DomainHash][]uint64)
+
+			ghostdagData := make([]*externalapi.BlockGHOSTDAGDataHashPair, 0)
```

Comment:
Why did you leave out the capacity here?

Reply:
Because it's hard to estimate the capacity